<launch>
    <!-- Nodo de local grid map -->
    <node name="local_grid_map" pkg="local_grid_map" type="local_grid_map_node" output="screen">
        <!-- ParÃ¡metros que pueden ser ajustados -->
        <param name="use_lidar" value="true" />
        <param name="use_point_cloud" value="true" />
        <param name="min_x" value="-0.5" />
        <param name="max_x" value="3.5" />
        <param name="min_y" value="-2.0" />
        <param name="max_y" value="2.0" />
        <param name="min_z" value="0.05" />
        <param name="max_z" value="1.50" />
        <param name="footprint_min_x" value="-0.3" />
        <param name="footprint_max_x" value="0.3" />
        <param name="footprint_min_y" value="-0.3" />
        <param name="footprint_max_y" value="0.3" />
        <param name="no_sensor_data_timeout" value="0.5" />
        <param name="cloud_points_threshold" value="100" />
        <param name="cloud_downsampling" value="9" />
        <param name="lidar_points_threshold" value="20" />
        <param name="lidar_downsampling" value="1" />
        <param name="inflation_radius" value="0.1" />
        <param name="cost_radius" value="0.1" />
        <param name="point_cloud_topic" value="/hsrb/head_rgbd_sensor/depth_registered/rectified_points" />
        <param name="laser_scan_topic" value="/hsrb/base_scan" />
    </node>
</launch>
